app/controllers/workorders_controller.rb

```diff
@@ -1,4 +1,17 @@
 class WorkordersController < ApplicationController
+  def index
+    @workorder = Workorder.all
+  end
+
+  def update
+    @workorder = Workorder.find(params[:id])
+    if params[:status] != ''
```

Comment:
we try a second time but it doesn't work. I think we should keep it like this. thx